src/pull_automerge/core.clj

```diff
@@ -94,17 +94,30 @@
   (def org "ai-labs-team")
   (def repo "axiom-platform")
   (def label "Automerge")
+  (def priority "High%20Priority")
   (def options (generate-options token))
   (check-rate-limit options)
-  
+ 
   ; ***************************************************************************
-  ; GET OPEN LABELED ISSUES
+  ; GET OPEN LABELED ISSUES (HIGH PRIORITY FIRST)
   ; ***************************************************************************
-  (def pulls-result @(http/get 
-    (get-pull-search-url org repo label)
+  (def pulls-result @(http/get
+    (get-pull-search-url org repo label priority)
     options))
-  (println "Retrieve automerge issues status: " (pulls-result :status))
-  (def pulls (parse-string (pulls-result :body) true))
+  (println "Retrieve automerge + high priority issues status: " (pulls-result :status))
+  (def pulls (parse-string (pulls-result :body) true)) 
+ 
+  ; ***************************************************************************
+  ; GET OPEN LABELED ISSUES
+  ; ***************************************************************************
+  (if (= 0 (count pulls))
+    (do 
+      (def pulls-result @(http/get 
```

Comment:
This guy can probably be pulled into its own function with varargs to DRY this a bit